qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -21,18 +21,11 @@ class iTestChannel(InstrumentChannel):
     
     def __init__(self, parent: Instrument,
                        name: str,
-                       chan_num: int,
-                       synchronous_enable: bool=True,
-                       synchronous_delay: float=1,
-                       synchronous_threshold: float=1e-5) -> None:
+                       chan_num: int) -> None:
         """
         Args:
             parent: The instrument to which the channel is attached.
             name: The name of the channel.
-            chan_num: The number of the channel in question
```

Comment:
Perhaps chan_num explanation should stay :)

Reply:
oops